Source/WebCore/dom/Document.h

```diff
@@ -1666,7 +1666,7 @@ class Document
     const FrameSelection& selection() const { return m_selection; }
         
     void setFragmentDirective(const String& fragmentDirective) { m_fragmentDirective = fragmentDirective; }
-    const String& fragmentDirective() const { return m_fragmentDirective; }
+    const String fragmentDirective() const { return m_fragmentDirective; }
```

Comment:
I don't think this is a good change.  We only need to return a reference in a const getter.